registration/include/pcl/registration/ndt.h

```diff
@@ -168,13 +168,26 @@ class NormalDistributionsTransform : public Registration<PointSource, PointTarge
     outlier_ratio_ = outlier_ratio;
   }
 
+  /** \brief Get the registration alignment likelihood.
+   * \return transformation likelihood
+   */
+  inline double
+  getTransformationLikelihood() const
+  {
+    return trans_likelihood_;
+  }
+
   /** \brief Get the registration alignment probability.
    * \return transformation probability
    */
+  PCL_DEPRECATED(1,
+                 16,
+                 "The method `getTransformationProbability` was renamed to "
+                 "`getTransformationLikelihood`.")
```

Comment:
Would present tense make more sense?

Reply:
Please correct me if I'm wrong, I understand it like this: The method _was_ renamed and therefore it _is_ now deprecated.